test/powershell/engine/ResourceValidation/TestRunner.ps1

```diff
@@ -0,0 +1,57 @@
+function Test-ResourceStrings

```

Comment:
Would it be better to put this into an existing module or a new test module?

Reply:
i wanted to do it this way because I do *not* want to make this a general function - we shouldn't be testing internal data structures, but there's no way to do this test without it.

Reply:
ok